frontend-ssr/components/notifications-tooltip/NotificationsTooltip.component.tsx

```diff
@@ -0,0 +1,221 @@
+import React from 'react';
```

Comment:
This should be named `NotificationTooltip` - just like the folder to keep the same convention as the other components.